src/commands/capabilities/translations/create.ts

```diff
@@ -0,0 +1,54 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { type CapabilityLocalization } from '@smartthings/core-sdk'
+
+import { apiDocsURL } from '../../../lib/command/api-command.js'
+import {
+	apiOrganizationCommand,
+	apiOrganizationCommandBuilder,
+	type APIOrganizationCommandFlags,
+} from '../../../lib/command/api-organization-command.js'
+import { capabilityIdBuilder, type CapabilityIdInputFlags } from '../../../lib/command/capability-flags.js'
+import {
+	inputAndOutputItem,
+	inputAndOutputItemBuilder,
+	type InputAndOutputItemFlags,
+} from '../../../lib/command/input-and-output-item.js'
+import { chooseCapability } from '../../../lib/command/util/capabilities-choose.js'
+import { buildTableOutput } from '../../../lib/command/util/capabilities-translations-table.js'
+
+
+export type CommandArgs =
+	& APIOrganizationCommandFlags
+	& InputAndOutputItemFlags
+	& CapabilityIdInputFlags
+
+const command = 'capabilities:translations:create [id]'
+
+const describe = 'create a capability translation'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	inputAndOutputItemBuilder(capabilityIdBuilder(apiOrganizationCommandBuilder(yargs)))
+		.example([
+			[
+				'$0 capabilities:translations:create custom1.outputModulation -i en.yaml',
+				'create a translation for the capability "custom1.outputModulation" from the definition in "en.yaml"',
+			],
+			[
+				'$0 capabilities:translations:create capabilities:translations:create -i en.yaml',
```

Comment:
I don't think `capabilities:translations:create` is meant to be repeated here.

Reply:
Heh, oops. Fixed.